MinionMovies/DetailsView/DetailsViewWorker.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  DetailsViewWorker.swift
+//  MinionMovies
+//
+//  Created by Vitor Costa on 14/05/20.
+//  Copyright © 2020 Vitor Costa. All rights reserved.
+//
+
+import Foundation
+
+protocol DetailsViewWorkerProtocol {
+    var dataProvider: DetailsViewDataProviderProtocol? { get set }
+    
+    func add(item: FavMovieDB)
+    func deleteMovie(with id: String)
+    func checkMovie(with id: String) -> Bool
```

Comment:
Não diz o que checa aqui tbm e se for pra checar sem existe tu pode usar a função de fetch